save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/diktat/DiktatDemoTool.kt

```diff
@@ -0,0 +1,17 @@
+package com.saveourtool.save.demo.diktat
+
+/**
+ * Enum that represents possible tool names in diktat-demo
+ */
+enum class DiktatDemoTool {
```

Comment:
May be some more general name? Why do we need `diktat` prefix?

Reply:
Cause this demo is responsible for diktat and ktlint. Our diktat-demo can run two tools.